### ISSUE TITLE
Reformat sp_DatabaseRestore

### ISSUE THREAD

**Issue by ShawnCrocker** [BrentOzarULTD/SQL-Server-First-Responder-Kit#1678]
A minor change but doing this in a feature change will generate too many changes in the pull request

- fix line endings to Windows CRLF (currently *nix LF)
- trim trailing spaces
- indent code to follow nesting level
- remove some blank lines
- ...and whatever I see

**Comment by BrentOzar** [comment]
Normally we go all ☹️  when people try that, but it makes sense here since you're doing a lot of maintenance on it. I know I like maintaining stuff more when it's formatted the way I like too, hahaha. I don't think anybody else has active work pending on this proc, so you can reformat it and do a PR whenever works for you. Thanks sir!

**Comment by BlitzErik** [comment]
@ShawnCrocker closing this for now. If you wanna pick it back up anytime, reopen or start a new issue.
